tests/default.nf.test.snap

```diff
@@ -145,5 +1297,147 @@
             "nextflow": "25.04.6"
         },
         "timestamp": "2025-09-04T09:54:04.87601553"
+    },
+    "nonpareil": {
+        "content": [
+            [
+                "2612_ERR5766176.npc",
+                "2612_ERR5766176.npl",
+                "2612_ERR5766176.npo",
+                "2612_ERR5766176.png",
+                "2612_ERR5766176_B.npc",
+                "2612_ERR5766176_B.npl",
+                "2612_ERR5766176_B.npo",
+                "2612_ERR5766176_B.png",
+                "2612_ERR5766180.npc",
+                "2612_ERR5766180.npl",
+                "2612_ERR5766180.npo",
+                "2612_ERR5766180.png",
+                "2613_ERR5766181.npc",
+                "2613_ERR5766181.npl",
+                "2613_ERR5766181.npo",
+                "2613_ERR5766181.png",
+                "nonpareil_all_samples.csv",
+                "nonpareil_all_samples.json",
+                "nonpareil_all_samples.pdf",
+                "nonpareil_all_samples.tsv",
+                "nonpareil_all_samples_mqc.png"
+            ],
+            [
+                
+            ],
+            true,
+            true,
+            true,
+            true
+        ],
+        "meta": {
+            "nf-test": "0.9.3",
+            "nextflow": "25.04.8"
+        },
+        "timestamp": "2025-11-05T15:32:06.208442425"
+    },
+    "centrifuge": {
+        "content": [
+            [
+                "centrifuge_db3_combined_reports.txt",
+                "db3/2612_db3.centrifuge.mapped.fastq.gz",
+                "db3/2612_db3.centrifuge.report.txt",
+                "db3/2612_db3.centrifuge.results.txt",
+                "db3/2612_db3.centrifuge.txt",
+                "db3/2612_db3.centrifuge.unmapped.fastq.gz",
+                "db3/2613_db3.centrifuge.mapped.fastq.gz",
+                "db3/2613_db3.centrifuge.report.txt",
+                "db3/2613_db3.centrifuge.results.txt",
+                "db3/2613_db3.centrifuge.txt",
+                "db3/2613_db3.centrifuge.unmapped.fastq.gz"
+            ],
+            true,
+            true,
+            [
+                "centrifuge_db3_combined_reports.txt",
```

Comment:
These should have md5sums as they are in your `stable_content_centrifuge` variable, why are they missing 🤔